x11-libs/libxinerama/libxinerama-1.1.4.recipe

```diff
@@ -19,37 +19,40 @@ libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 
 PROVIDES="
 	libxinerama$secondaryArchSuffix = $portVersion
-	lib:libxinerama$secondaryArchSuffix = $libVersionCompat
+	lib:libXinerama$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libxext$secondaryArchSuffix
 	lib:libX11$secondaryArchSuffix
+	lib:libXau$secondaryArchSuffix
+	lib:libXcb$secondaryArchSuffix
+	lib:libXdmcp$secondaryArchSuffix
+	lib:libXext$secondaryArchSuffix
```

Comment:
For these  cases I've used "libX*" instead of "libx*" looks more consistent to the sonames it provides ...